package.json

```diff
@@ -35,7 +35,8 @@
       "temperature": "bricklets/Temperature.js",
       "digiin": "bricklets/IndDigital4In.js",
       "analogout": "bricklets/AnalogOut.js",
-      "indanalogout": "bricklets/IndAnalogOut.js"
+      "indanalogout": "bricklets/indAnalogOut.js",
```

Comment:
This needs to be bricklets/IndAnalogOut.js not bricklets/indAnalogOut.js.  This is fixed in the latest head stream, please make sure you pull it in.